src/Juvix/Compiler/Core/Transformation/ComputeTypeInfo.hs

```diff
@@ -61,7 +61,7 @@ computeNodeTypeInfo tab = umapL go
         let ci = fromJust $ HashMap.lookup _constrTag (tab ^. infoConstructors)
             ii = fromJust $ HashMap.lookup (ci ^. constructorInductive) (tab ^. infoInductives)
          in case ii ^. inductiveBuiltin of
-              Just BuiltinBool ->
+              Just (BuiltinTypeInductive BuiltinBool) ->
```

Comment:
what about `BuiltinNat`?

Reply:
It's supposed to have been removed at this point. Besides, it's an ordinary inductive type with no special representation, so the default case would work for it.